### DISCUSSION TITLE
Want to implement rectangle selection without zoom.

### DISCUSSION THREAD

**Discussion by kaushikdeo** [chartjs/chartjs-plugin-zoom#866] [Q&A] [answered]
firstly awesome job on the plugin. Its really helpful.

Below is the functionality for which i need some help and/or clarifications

1. Should be able to drag and draw a selection rectangle ( this is happening)
2. once the rectangle is drawn i want to know the start and end of the xaxis points
3. I dont want to zoom the chart but use the xaxis start and end points to do some calculations
4. I also want the selection rectangle to persist
5. show the calculations in a box

Can someone guide me on this?

**Comment by MauricioFauth** [comment, marked as answer]
Hi @kaushikdeo,

Maybe these other plugins are more suitable for your needs:
- https://www.npmjs.com/package/chartjs-plugin-boxselect
- https://www.npmjs.com/package/chartjs-plugin-dragzone
